addons/mail/static/src/models/file_uploader.js

```diff
@@ -145,13 +146,17 @@ registerModel({
                         return;
                     }
                     const attachment = this._onAttachmentUploaded({ attachmentData, composer, thread });
-                    attachments.push(attachment);
+                    uploadedAttachments.push(attachment);
                 } catch (e) {
                     if (e.name !== 'AbortError') {
                         throw e;
                     }
                 }
             }
+            for (const i in uploadedAttachments) {
+                const attachment = (composer || thread).messaging.models['Attachment'].insert(uploadedAttachments[i]);
```

Comment:
Not a huge deal, but could use `const data of uploadedAttachments` so that you don't need `[i]`.

_(FYI, `of` loops on array value immediately, whereas `in` loops on keys of object/array)_

Reply:
Done